pkg/kudoctl/cmd/repo_index.go

```diff
@@ -151,7 +151,7 @@ func (ri *repoIndexCmd) run() error {
 }
 
 func merge(index *repo.IndexFile, mergeIndex *repo.IndexFile) {
-	// index is the master, any dups in the merged in index will have what is local replace those entries
+	// index is the main, any dups in the merged in index will have what is local replace those entries
```

Comment:
hmm... not sure this makes sense... but I don't know the best solution... `main` doesn't make sense.   master here means master copy...  after re-reading.. this comment is (likely written by me) and it not great :(  AND it should be the line above func, not after.

`// index is the reference copy which will have non-duplicated entries added to it from the mergedIndex`